schema.graphql

```diff
@@ -32,6 +33,9 @@ type Tree @entity {
   id: ID!
   childOfTree: Tree
   linkedToHat: Hat
+  requestedLinkToTree: Tree
+  requestedLinkToAdminHat: Hat
```

Comment:
And here as well, though I think I'd simplify to "requestedLinkToHat" to mirror the "requestedLinkToTree" version